authors/apps/like_dislike/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from django.contrib import admin
+# from .views import (createpost, detail_post_view, postpreference)
+from . import views
+from .models import LikeDislike
+
+app_name = "like"
+
+urlpatterns = [
+    path('article/<slug>/like/', views.PreferenceView.as_view(
```

Comment:
inconsistencies from the article URLs you are using singular while articles app is in plurals. Fix them

Reply:
I've fixed this